lib/tasks/accounts.rake

```diff
@@ -0,0 +1,74 @@
+namespace :accounts do
+  desc "merge two user accounts using environment variables SRC=<username-1> DST=<username-2>"
+  task merge: [:connection] do
+    if !ENV['SRC'] || !ENV['DST']
+      STDERR.puts "Usage: SRC=<username-1> DST=<username-2> rake accounts:merge"
+      exit 1
+    end
+
+    require 'exercism/user'
+
+    src = User.where(username: ENV['SRC']).first
+    if !src
+      STDERR.puts "Cannot find user record for %s" % ENV['SRC']
+      exit 1
+    end
+
+    dst = User.where(username: ENV['DST']).first
+    if !dst
+      STDERR.puts "Cannot find user record for %s" % ENV['DST']
+      exit 1
+    end
+
+    require 'exercism/user_exercise'
+    require 'exercism/named'
+    require 'exercism/problem'
+    require 'exercism/submission'
+    require 'exercism/acl'
+
+    src.exercises.each do |src_exercise|
+      # sometimes people have done the same exercise on both accounts
+      target = UserExercise.where(language: src_exercise.language, slug: src_exercise.slug, user_id: dst.id).first
+      if !!target
+        # destination exists, we need to reset some stats on the old one
+        # so it stops showing up on the old profile
+        src_exercise.iteration_count = 0
+        src_exercise.save
+      else
+        # destination doesn't exist, so move the old exercise to the new user account
+        target = src_exercise
+        target.user_id = dst.id
+        target.save
+      end
+
+      src_exercise.submissions.each do |submission|
```

Comment:
I was thinking of doing SQL, but my brain was a bit fried so it was easier to make it work the brute-force way. For what it's worth, it is a task that will not be run very often, and then only on a single person's data, so I'm less worried about N+1 than if it were a hot code path.

Doing an `includes(:submissions)` would be an improvement. I'll use that.